src/js/disability-benefits/components/ClaimEstimationPage.jsx

```diff
@@ -16,7 +17,7 @@ export default class ClaimEstimationPage extends React.Component {
             <nav className="va-nav-breadcrumbs">
               <ul className="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
                 <li><Link to="your-claims">Your claims</Link></li>
-                <li><Link to={`your-claims/${this.props.params.id}/status`}>Your Disability Compensation Claim</Link></li>
+                <li><Link to={`your-claims/${this.props.params.id}/status`}>Your {getClaimType(this.props)} Claim</Link></li>
```

Comment:
This component doesn't currently receive the claim as a prop, I think you can get it from the store/state by adding
```
function mapStateToProps(state) {
  return {
    claim: state.disability.status.claimDetail.detail
  }
}

export default connect(mapStateToProps)(ClaimEstimationPage);
```
@jbalboni Can you confirm the above?